docs/guide-disappearance.md

```diff
@@ -43,6 +43,21 @@ test('movie title goes away', async () => {
 })
 ```
 
+The `waitForElementToBeRemoved` [async helper][async-api] function uses a
```

Comment:
Let's move this to be the first example in this section (before the `wait` example) because it's probably the better option.